app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -32,7 +32,7 @@ DEPEND=">=dev-lang/go-1.2
 RDEPEND=""
```

Comment:
Unrelated to the 1.3 question, but either add a conflicts with the old rocket name so it gets uninstalled or add an entry to coreos-overlay/profiles/updates to inform portage that the name of the installed package has changed.